src/figuresClasses.ts

```diff
@@ -1,11 +1,75 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea: () => number;
+}
+
+function areaRound(area: number): number {
+  return Math.floor(area * 100) / 100;
+}
+
+export class Triangle implements Figure {
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+    public shape: string = `triangle`,
+  ) {
```

Comment:
In TypeScript, when declaring a constructor parameter with an access modifier, it's not necessary to assign it manually to a property. The `color`, `a`, `b`, `c`, and `shape` properties will be automatically assigned. You can remove the manual assignments.